simphony_metadata.yml

```diff
@@ -16,7 +16,8 @@ CUDS_KEYS:
     parent:
     CUBA.UUID:
       scope: CUBA.SYSTEM
-    CUBA.DATA:
+    data:
```

Comment:
Well, it's not. The value for `data` is undefined in metadata. As you mentioned, it is a structure and would be problematic to define. @ahashibon Any coment on this?